docs/locales/de/LC_MESSAGES/index.po

```diff
@@ -0,0 +1,80 @@
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: BeeWare 0.3\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-01-06 18:04+0800\n"
+"PO-Revision-Date: 2024-01-06 20:03+0800\n"
+"Last-Translator: Russell Keith-Magee <russell@keith-magee.com>\n"
+"Language-Team: de <LL@li.org>\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Generated-By: Babel 2.14.0\n"
+"X-Generator: Poedit 3.4.2\n"
+
+#: ../../index.rst:3
+msgid "Welcome to the BeeWare Tutorial!"
+msgstr "Willkommen zum BeeWare-Tutorial!"
+
+#: ../../index.rst:5
+#, fuzzy
+msgid "**Write Python. Run Anywhere.**"
+msgstr "**Python schreiben. Überall ausführen.**"
+
+#: ../../index.rst:7
+#, fuzzy
+msgid "Welcome to BeeWare! In this tutorial, we're going to build an graphical user interface using Python, and deploy it as a desktop application, as a mobile application, and as a single page web app. We're also going to look at how you can use BeeWare tools to do some of the common tasks that you'll need to do as an app developer, such as testing your app."
+msgstr "Willkommen bei BeeWare! In diesem Tutorial werden wir eine grafische Benutzeroberfläche mit Python erstellen und sie als Desktop-Anwendung, als mobile Anwendung und als einseitige Webanwendung bereitstellen. Außerdem sehen wir uns an, wie Sie die BeeWare-Tools nutzen können, um einige der üblichen Aufgaben zu erledigen, die Sie als App-Entwickler benötigen, wie zum Beispiel das Testen Ihrer App."
```

Comment:
These sentences had become very long and therefore hard to understand.
My suggested translation deviates a tiny bit from the original while being much more readable in my opinion.

Lost meaning:
- "single page"

```suggestion
msgstr "Willkommen bei BeeWare! In diesem Tutorial werden wir eine grafische Benutzeroberfläche mit Python erstellen und sie als Desktop-, Mobil- und Web-Anwendung bereitstellen. Außerdem sehen wir uns an, wie Sie die BeeWare-Tools nutzen können, um übliche Aufgaben als App-Entwickler zu erledigen, wie zum Beispiel das Testen Ihrer App."
```